rts/lra-examples/coordinator-quarkus/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.jboss.narayana.quickstart.rts.lra</groupId>
+    <artifactId>lra-examples</artifactId>
+    <version>5.13.1.Final-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>lra-coordinator-quarkus</artifactId>
+  <packaging>jar</packaging>
+  <name>Quarkus LRA Coordinator</name>
+
+  <properties>
+
+    <failOnMissingWebXml>false</failOnMissingWebXml>
+    <narayana.nodename.property>CoreEnvironmentBean.nodeIdentifier</narayana.nodename.property>
+    <narayana.version>${project.version}</narayana.version>
+    <quarkus.http.port>8080</quarkus.http.port>
+    <surefire-plugin.version>2.22.0</surefire-plugin.version>
+    <version.quarkus>2.12.1.Final</version.quarkus>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>io.quarkus</groupId>
+        <artifactId>quarkus-bom</artifactId>
+        <version>${version.quarkus}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-rest-client</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-resteasy-jsonb</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-openapi</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-fault-tolerance</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.narayana.rts</groupId>
+      <artifactId>lra-coordinator-jar</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <finalName>lra-coordinator</finalName>
```

Comment:
I don't think we should override the final name, that combined with `<quarkus.package.type>uber-jar</quarkus.package.type` had something to do with the Rule "sources-staging" failures during the release.

Reply:
Thanks Michael. I am going to remove it then